Client/Frontend/Browser/TabManager.swift

```diff
@@ -118,5 +118,6 @@ class TabManager {
         }
         
         assertionFailure("Tab not in tabs list")
+        return -1
```

Comment:
I guess we could change the signature, yeah. But I didn't bother, because we will error out when we hit the assertion — this return is just there to make the compiler happy!